### PR TITLE
"Unhandled Rejection (TypeError): Cannot read property 'call' of undefined" when trying to change pages

### PR DESCRIPTION
**Pull request by Enalmada**
I am experiencing a fatal issue when trying to click links in an example with pages including less files.  I did this as a pull request so there would be an easily repeatable testcase available. (originally was working on this as a pull request to upgrade example to next 7 and hope to finish submitting as such  when issue fixed)
 
1)  Logs are full of "chunk styles [mini-css-extract-plugin] Conflicting order between..." statements:
image
There is some indication we could just [ignore the warnings](https://github.com/webpack-contrib/mini-css-extract-plugin/issues/250#issuecomment-421989979) if next.js can't do anything about this (but I couldn't figure it out).

2)  "Unhandled Rejection (TypeError): Cannot read property 'call' of undefined" when trying to change pages.
image

Note that this issue goes away after reload.  It also doesn't seem to happen in production build. Not using less at all seems to fix the problem.

### CODE REVIEW

File: examples/with-ant-design-less/package.json [hunk 1]
```diff
@@ -7,16 +7,12 @@
     "start": "next start"
   },
   "dependencies": {
-    "@zeit/next-less": "0.3.0",
+    "@zeit/next-less": "^1.0.0",
```
[the remark this answers is not in the document]
Comment (Enalmada) [reply examples/with-ant-design-less/package.json]: hmmm...that must have come out right after.  Will try now...

**Comment by popugang** [comment]
i have same problem with css-modules in sass. 

On SSR styles build, but i going on some page i have error "Unhandled Rejection (TypeError): Cannot read property 'call' of undefined"

**Comment by serzmerz** [comment]
i have the same issue with sass

**Comment by Yuripetusko** [comment]
Same issue with stylus `Cannot read property 'call' of undefined"` if `mini-css-extract-plugin` cannot work out css import order

**Comment by frizadiga** [comment]
i have the same issue with sass

**Comment by JiiB** [comment]
I have the same issues also using `@zeit/next-sass` 1.0.1

**Comment by JiiB** [comment]
@Enalmada any news on this issue? Were you able to find a solution for this problem? I'm still having trouble with that.

**Comment by Yuripetusko** [comment]
We switched back to styled-jsx for now for external stylus files. And slowly rewriting everything to Fela

**Comment by Enalmada** [comment]
@JiiB I am under the impression it is being worked on based on this chat: https://spectrum.chat/thread/2183fc55-236d-42cb-92b9-3ab10acc6303?m=MTUzODU2NDg2ODA2Mg==

I happened to workaround it by using reason-react...you can see an example of that [here](https://github.com/Enalmada/next-reason-boilerplate).  That project is a work in progress but I am posting it on the off chance it helps someone.

**Comment by Jero786** [comment]
I have the same issue here! please, some kind of solution at least some workaround? I also want to add that this error only happen to me in Chrome.

**Comment by skywickenden** [comment]
My work around for this has been to move the style imports from the top of the file to the render function.  Seems to manage to keep them separate this way.

```
// import styles from './Foo.scss';

class Foo extends Component {
...
render () {
  const styles = require('./Foo.scss');
  ...
}

...

```

**Comment by JiiB** [comment]
@dmytro-lymarenko 
Unfortunetaly not, I ended up using [styled-components](https://www.styled-components.com/). Luckly I was at the start of development, so there was not too much to refactor

**Comment by Jero786** [comment]
I'm still have the same issue, it's really stress for me. 

## System information
 - OS: MacOs 10.13.6
 - Browser chrome
- Package dependencies: 
```
 "@zeit/next-sass": "^1.0.2-canary.0",
    "babel-polyfill": "^6.26.0",
    "classnames": "^2.2.6",
    "cross-fetch": "^2.2.2",
    "express": "^4.16.4",
    "fs": "0.0.1-security",
    "immutable": "^3.8.2",
    "lodash": "^4.17.10",
    "next": "^7.0.2-canary.21",
    "next-redux-wrapper": "^2.0.0",
    "node-sass": "4.9.4",
    "prop-types": "^15.6.2",
    "react": "^16.4.2",
    "react-dom": "^16.4.2",
    "react-notification-system": "^0.2.17",
    "react-redux": "^5.0.7",
    "redux": "^4.0.0",
    "redux-devtools-extension": "^2.13.5",
    "redux-immutable": "^4.0.0",
    "redux-thunk": "^2.3.0"
```

my dev dependencies:
```
"devDependencies": {
    "babel-core": "7.0.0-bridge.0",
    "babel-eslint": "^8.2.6",
    "babel-jest": "22.4.3",
    "babel-plugin-module-resolver": "^3.1.1",
    "chai": "^4.1.2",
    "enzyme": "3.2.0",
    "enzyme-adapter-react-16": "1.5.0",
    "eslint": "^4.19.1",
    "eslint-config-airbnb": "^17.0.0",
    "eslint-plugin-import": "^2.13.0",
    "eslint-plugin-jsx-a11y": "^6.1.1",
    "eslint-plugin-react": "^7.10.0",
    "husky": "^0.14.3",
    "jest": "22.0.1",
    "lint-staged": "^7.3.0",
    "prettier-eslint": "^8.8.2",
    "react-addons-test-utils": "15.6.2",
    "react-test-renderer": "16.2.0",
    "sinon": "^6.1.5",
    "superagent": "^4.0.0-beta.5",
    "supertest": "^3.3.0"
  }
```

48554964-8950bc00-e8be-11e8-9d0a-a77eaaa4f3da

**Comment by JiiB** [comment]
@Jero786 have you tried upgrading to the current `react` and `react-dom` version **16.6.3**?
Also make sure that you are not importing any empty `.scss` files in your react components

**Comment by Jero786** [comment]
> @Jero786 have you tried upgrading to the current `react` and `react-dom` version **16.6.3**?
> Also make sure that you are not importing any empty `.scss` files in your react components

HI @JiiB ! thanks for responde me so quickly, I already check and indeed I was imported an empty scss, but now the error message change! let me show you guys.. 
screen shot 2018-11-19 at 11 17 47 am

Do you have some idea about that? Thanks in adcance!

**Comment by pitops** [comment]
Is there any update on this? I am having the same problem as well. Using .less files and .css imports

**Comment by samhenri** [comment]
I'm having the same issue with next-sass. Here's my current next config:

```
const withSass = require('@zeit/next-sass');

module.exports = withSass({
  cssModules: false,
  webpack: (config, { dev }) => {
    config.module.rules.push(
      {
        test: /\.js$/,
        loader: 'eslint-loader',
        exclude: /node-modules/,
        enforce: 'pre',
        options: {
          emitWarning: true
        }
      },
      {
        test: /\.(config)$/,
        loader: 'file-loader',
        options: {
          name: '[path][name].[ext]',
        }
      },
      {
        test: /\.(png|ttf|eot|wtf|jpg|svg|gif)$/,
        use: [
          {
            loader: 'url-loader',
            options: {}
          }
        ]
      },
    );

    const StyleLintPlugin = require('stylelint-webpack-plugin');

    config.plugins.push(
      new StyleLintPlugin({
        emitErrors: false,
        quiet: false,
      }),
    );

    return config;
  }
});
```
Tried with `node-sass 0.2.1-canary.2` and `next v7.0.2-canary.41`

**Comment by timneutkens** [comment]
I believe this PR was superseded by another PR considering there are merge conflicts. I'll close it for now as the build also failed

**Comment by Enalmada** [comment]
@timneutkens Can you please link to the PR you believe supersedes this?

**Comment by afuggini** [comment]
Using `"@zeit/next-less": "^1.0.2-canary.2"` worked for me, thanks!

**Comment by afuggini** [comment]
I could fix it like this: https://github.com/zeit/next-plugins/issues/282#issuecomment-446379427

**Pull request closed without merging**